app/src/main/java/com/randomappsinc/studentpicker/listpage/ListActivity.java

```diff
@@ -48,6 +51,9 @@ protected void onCreate(Bundle savedInstanceState) {
                 listName,
                 listTabTitles);
         nameListPager.setAdapter(listTabsAdapter);
+        if (getIntent().getBooleanExtra(START_ON_EDIT_PAGE,false)){
```

Comment:
nit: Small spacing issue
```suggestion
        if (getIntent().getBooleanExtra(START_ON_EDIT_PAGE, false)) {
```